### PR TITLE
Feature/loading

### PR DESCRIPTION
**Pull request by leticiawanderley**
#### What is the purpose of this pull request?
* Add loading svg icon
* Add 'no purchase permission' modal
* Add home page

#### Screenshots or example usage
home page

modal

#### Types of changes
* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    intl: intlShape.isRequired,
+  }
+
+  translate = id => this.props.intl.formatMessage({ id: `extensions.${id}` })
+
+  render() {
+    return (
+      <div className="w-100 h-100">
+        <div className="bg-serious-black pt9 pb10">
+          <div className="ml5-s ml7-ns mt6 white">
```
Comment (kevinch) [inline react/HomePage.js]: Are all those 3 container `div`s required?

File: react/components/Header.js [hunk 2]
```diff
@@ -65,20 +72,27 @@ class Header extends Component {
           </div>
         </div>
         <div className="tr-ns flex items-center">
-          {logged ? (
+          {!logged ? (
             <Profile
               name="Bill Zoo"
```
Comment (kevinch) [inline react/components/Header.js]: Hard coded text?

File: react/components/NoPermissionModal.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    const { onClose } = this.props
+    return (
+      <div className="near-black">
+        <div className="b f4 mb6-s mb5-ns">{this.translate('oops')}.</div>
+        <div className="f5 mb6-s mb8-ns">
+          <FormattedMessage
+            id="extensions.noPermission"
+            values={{
+              purchasePermission: (
+                <a className="link b rebel-pink">
```
Comment (kevinch) [inline react/components/NoPermissionModal.js]: For a link with i18n i'd go for 
```
<a href="">
    < FormattedMessage />
</a>
```
Easier to read in my opinion.

File: react/components/NoPermissionModal.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                  {this.translate('purchasePermission')}
+                </a>
+              ),
+            }}
+          />
+        </div>
+        <div className="flex justify-center-s justify-end-ns tr">
+          <div className="w-40-s w-20-ns bg-rebel-pink br2">
+            <Button onClick={onClose} block>
```
Comment (kevinch) [inline react/components/NoPermissionModal.js]: `onClose` does not exist.

Comment (leticiawanderley) [reply react/components/NoPermissionModal.js]: It is one of the `props`

File: react/components/Header.js [hunk 5]
```diff
@@ -65,20 +72,27 @@ class Header extends Component {
...
               </Button>
             </div>
           )}
         </div>
+        <Modal
+          centered
+          isOpen={this.state.isModalOpen}
+          onClose={this.handleModal}
+        >
+          <NoPermissionModal onClose={this.handleModal} />
```
Comment (klzns) [inline react/components/Header.js]: Pq o `NoPermissionModal` nao tem `Modal` em sua implementação?

File: react/components/NoPermissionModal.js (hunk 4 above)
Comment (kevinch) [reply react/components/NoPermissionModal.js]: My bad.

Comment (leticiawanderley) [reply react/components/NoPermissionModal.js]: No problem 😉

File: react/components/Header.js (hunk 5 above)
Comment (leticiawanderley) [reply react/components/Header.js]: Tem razão, mudei

**Pull request merged**